learning_journal/templates/layout.jinja2

```diff
@@ -19,12 +19,7 @@
             </div>
```

Comment:
I can't comment above this line, but a few notes...

On line 4: you should be using `request.static_url` to call in your stylesheets.
On lines 13-16: you should be using `request.route_url('route_name')` for your nav links. Also why are there nav links for `detail` and `edit`? Shouldn't those only apply to specific entries, denoted by ID?